flow/order/list.yaml

```
@@ -1,57 +1,164 @@
 steps:
-  # In:
-  #  user.id (str) - id of current user passed in from page
-  # Out:
-  #  flow.order_id (str) - id of selected order
-  - type: component.order.list
-    user_id: (@ user.user_id )
+  - (choose filter)
+  - flow_set:
+      fulfillment_status: any
+      financial_status: any
+      order_status: any
+
+  - ask: Please choose the order filter
+    buttons:
+      - text: Fulfillment status
+        action:
+          jump: fulfillment status
+      - text: Financial status
+        action:
+          jump: financial status
+      - text: Order status
+        action:
+          jump: order status
+      - text: See all orders
+        result: any
+  - flow_set: order_status
+  - jump: orders query
```

Can be collapsed like this:

```suggestion
        action:
          jump: orders query
          data:
            order_status: any
```

---

Actually, looks like `order_status` will always be "any", so no need to set that